com.unity.render-pipelines.high-definition/Editor/ShaderGraph/HDTarget.cs

```diff
@@ -541,6 +541,15 @@ static class CoreKeywords
             { CoreKeywordDescriptors.AlphaTest, new FieldCondition(Fields.AlphaTest, true) },
         };
 
+        public static KeywordCollection HDBaseNoCrossFade = new KeywordCollection
```

Comment:
poking @Kink3d and @alelievr about this PR that affect shader graph and have been merged into HDRP/staging